prow/oss/config.yaml

```diff
@@ -263,6 +263,9 @@ presets:
   - name: docker-graph
     mountPath: /docker-graph
 
-pubsub_subscriptions:
-  "prow-build-looker-github":
-  - "projects/prow-build-looker-github/subscriptions/prow-test-sub"
+pubsub_triggers:
+  - allowed_clusters:
+      - "prow-build-looker-github"
```

Comment:
The alias for the cluster should be https://github.com/GoogleCloudPlatform/oss-test-infra/blob/0d7fff143fec438a6dd49dac14b3ac2283235f42/prow/oss/config.yaml#L97